### PR TITLE
fix: remove upper bound for node engine semver

### PR DESCRIPTION
**Pull request by pimlie**
Resolves: #48 

/cc @tinovyatkin

### CODE REVIEW

**Comment by readeral** [comment]
Interesting node-fetch changed the engine requirements to be `>=10.17.0` but with the linting issues, maybe the more specific version you've landed on is required by fetch-blob?

**Comment by tinovyatkin** [comment]
What do you see as a downside of  just ‘>=10.17’?

**Comment by tinovyatkin** [comment]
@xxczaki It's a serious bug, we will need to release a patch version after merging this (as it may preventing install of `node-fetch@3.beta7` via `yarn` on Node LTS)

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
         "node-fetch"
     ],
     "engines": {
-        "node": "^10.17.0"
+        "node": "^10.17.0 || >=12.3.0"
```
Comment (tinovyatkin) [inline package.json:23]:
```suggestion
        "node": ">=10.17"
```

**Comment by xxczaki** [comment]
@tinovyatkin We will also need to release a new beta version of `node-fetch` with the updated `fetch-blob` dependency (maybe something like `3.0.0-beta.7-fixnodeversion`), won't we?

**Comment by tinovyatkin** [comment]
@xxczaki No, `node-fetch` is fine, we have caret there, so, it will pick up 2.0.1 automatically. We should, however, deprecate 2.0.0 of `fetch-blob`

**Comment by pimlie** [comment]
@tinovyatkin See the first lint error, it said: `The stream.Readable.from is not supported until Node.js 12.3.0 (backported: ^10.17.0).`

File: package.json (hunk 1 above)
Comment (pimlie) [reply package.json:23]: This is what I added at first btw, but changed it due to the lint error. Not sure if v11 also supports it, I trusted the lint error to specify the correct minimum version

**Comment by xxczaki** [comment]
cc @tinovyatkin @pimlie

Released `fetch-blob@2.0.1` and deprecated `fetch-blob@2.0.0`.

**Comment by tinovyatkin** [comment]
@pimlie Thank you for your contribution!

**Pull request merged**